### ISSUE TITLE
RELEASES directory inflates the repository

### ISSUE THREAD

**Issue by t-8ch** [ArkoG/ARKOpack#10]
Keeping releases inside the repository will inflate the repository.
As git will not be able to compress them very well the repository will grow by 80MB per release.

**Comment by zijistark** [comment]
indeed, it's very bad practice to put large compressed files into a Git repository.  Git was not designed for that, as it's a version control system which stores all of its data as, in simple terms, changes from commit to commit. when a big ZIP file that can't be compressed (because it already is) changes, most of the binary ZIP file will be considered changed by Git each time they're added or updated, and the data will be replicated each time.

CPRplus unfortunately does this w/ their password-protected RAR of [modified] DLC assets, and by now, I can't even setup some of my servers to live-track their repository, because the disk usage is enormous (all that history of tiny little binary file changes that result in massive diffs).  Whenever I need to pull CPRplus for a release, I actually have to just let my machine (& GitHubs') chug & download several GB which it then needs to piece back together, taking more time, and then I can come back to it in 15-20mins — assuming my 250mbps network connection was performing above average and that I'm using very fast SSDs, which I am.

I'd recommend simply hosting the release ZIPs elsewhere, such as MediaFire (or whatever else that's free and convenient), and then simply link to them from your GitHub repository's `README.md` file -- basically the homepage of formatted text when you first navigate to the repository, under the list of files/directories.

**Comment by t-8ch** [comment]
The releases are already in GitHubs "releases" at https://github.com/ArkoG/ARKOpack/releases, so it should be just fine as it is.

**Comment by zijistark** [comment]
:+1: :1st_place_medal: :100: :star2: :rainbow:
